vald/start.go

```diff
@@ -287,7 +292,12 @@ func listen(clientCtx sdkClient.Context, txf tx.Factory, axelarCfg config.ValdCo
 	}
 
 	mgr.AddJobs(js...)
-	<-mgr.Done()
+	select {
+	case <-mgr.Done():
+		return
+	case <-blockTimeout.Done():
+		panic("no new blocks discovered, is the chain halted?")
```

Comment:
```suggestion
		panic("no new blocks received from the node")
```
since the bug occurs outside chain halt, maybe we should avoid the scary looking message? validators have other monitoring in place for chain halts anyways.